syntax = "proto3";

package robot;

service RobotServer {

    rpc moveForward (RobotRequest) returns (RobotReply) {}
    rpc moveBackward (RobotRequest) returns (RobotReply) {}
    rpc stopMove (RobotRequest) returns (RobotReply) {}
}

message RobotRequest {
    int32 id = 1;
}

message RobotReply {
    
}
